airflow/cli/cli_parser.py

```diff
@@ -905,6 +905,13 @@ def string_lower_type(val):
     help="The hostname of job(s) that will be checked.",
 )
 
+ARG_JOB_HOSTNAME_CALLABLE_FILTER = Arg(
+    ("--local",),
+    action='store_true',
+    help="If passed, this command will use the `hostname_callable` function"
+    "defined in Airflow configuration.",
```

Comment:
```suggestion
    help="If passed, this command will only show jobs from the local host "
    "(those with a hostname matching what `hostname_callable` returns).",
```

I think this reads a little better?